src/App.asm

```diff
@@ -1,6 +1,69 @@
 .data
+      
+       txt1: .asciiz " Enter total number of students: " 
+       txt2: .asciiz "Enter student score: "
+       txt3: .asciiz "Highest score is " 
+       x: .word 0
+       top: .word 0
+       sco: .word 0
+       stuN: .word 0
+       
+.globl main
 
+.text 
 
-.globl main
+    main: 
+               li $v0,4
+               la $a0,txt1
+               syscall 
+               
+               li $v0,5
+               syscall 
+               
+               move $t0,$v0
+               sw $t0,stuN
+               lw $t1,x
+               
+loop1:
+               blt $t1,$t0,loop2
```

Comment:
variables (`i` and `count`) used in while loop condition must be loaded from memory before such as:
```asm
...
lw  $t1, i
lw  $t0. count
blt $t1, t0, loop2
...
```